### PR TITLE
Samples version update

### PR DESCRIPTION
**Pull request by rokulka**
Samples version update to 4.4.0

### CODE REVIEW

File: experimental/qnamaker-activelearning/csharp_dotnetcore/qnamaker-activelearning-bot.csproj [hunk 1]
```diff
@@ -13,13 +13,13 @@
...
     <PackageReference Include="Microsoft.AspNetCore" Version="2.1.3" />
     <PackageReference Include="Microsoft.AspNetCore.All" Version="2.0.8" />
-    <PackageReference Include="Microsoft.Bot.Builder" Version="4.2.0" />
-    <PackageReference Include="Microsoft.Bot.Builder.AI.QnA" Version="4.2.0" />
-    <PackageReference Include="Microsoft.Bot.Builder.Dialogs" Version="4.2.0" />
-    <PackageReference Include="Microsoft.Bot.Builder.Integration.AspNet.Core" Version="4.2.0" />
-    <PackageReference Include="Microsoft.Bot.Configuration" Version="4.2.0" />
-    <PackageReference Include="Microsoft.Bot.Connector" Version="4.2.0" />
-    <PackageReference Include="Microsoft.Bot.Schema" Version="4.2.0" />
+    <PackageReference Include="Microsoft.Bot.Builder" Version="4.4.0" />
```
Comment (johnataylor) [inline experimental/qnamaker-activelearning/csharp_dotnetcore/qnamaker-activelearning-bot.csproj]: we're up to 4.4.3 now...

https://www.nuget.org/packages/Microsoft.Bot.Builder/4.4.3

File: experimental/qnamaker-support/csharp_dotnetcore/qnamaker-support-bot.csproj [hunk 2]
```diff
@@ -18,16 +18,16 @@
...
-    <PackageReference Include="Microsoft.Bot.Builder.AI.Luis" Version="4.2.0" />
-    <PackageReference Include="Microsoft.Bot.Builder.AI.QnA" Version="4.2.0" />
-    <PackageReference Include="Microsoft.Bot.Builder.Azure" Version="4.2.0" />
-    <PackageReference Include="Microsoft.Bot.Builder.Dialogs" Version="4.2.0" />
-    <PackageReference Include="Microsoft.Bot.Builder.Integration.ApplicationInsights.Core" Version="4.2.0" />
-    <PackageReference Include="Microsoft.Bot.Builder.Integration.AspNet.Core" Version="4.2.0" />
-    <PackageReference Include="Microsoft.Bot.Configuration" Version="4.2.0" />
-    <PackageReference Include="Microsoft.Bot.Connector" Version="4.2.0" />
-    <PackageReference Include="Microsoft.Bot.Schema" Version="4.2.0" />
+    <PackageReference Include="Microsoft.Bot.Builder" Version="4.4.0" />
```
Comment (johnataylor) [inline experimental/qnamaker-support/csharp_dotnetcore/qnamaker-support-bot.csproj]: same comment... in C# we're up to 4.4.3

File: experimental/qnamaker-activelearning/csharp_dotnetcore/qnamaker-activelearning-bot.csproj (hunk 1 above)
Comment (rokulka) [reply experimental/qnamaker-activelearning/csharp_dotnetcore/qnamaker-activelearning-bot.csproj]: done.

File: experimental/qnamaker-support/csharp_dotnetcore/qnamaker-support-bot.csproj (hunk 2 above)
Comment (sgellock) [reply experimental/qnamaker-support/csharp_dotnetcore/qnamaker-support-bot.csproj]: All Microsoft.Bot.* assemblies need to be updated to 4.4.3 please.

Comment (rokulka) [reply experimental/qnamaker-support/csharp_dotnetcore/qnamaker-support-bot.csproj]: They have been updated, this comments shows an outdated snapshot.

**Pull request merged**